infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryDao.kt

```diff
@@ -122,8 +120,10 @@ class GeometryDao @Autowired constructor(
             "direction_unit" to plan.units.directionUnit.name,
             "srid" to plan.units.coordinateSystemSrid?.code,
             "coordinate_system_name" to plan.units.coordinateSystemName,
-            "polygon_string" to plan.getBoundingPolygonPoints(kkjToEtrsTriangulationNetwork)
-                .let { p -> if (p.isEmpty()) null else create2DPolygonString(p) },
+            "polygon_string" to
+                    if (!boundingBoxInLayoutCoordinates.isNullOrEmpty())
+                        create2DPolygonString(boundingBoxInLayoutCoordinates)
+                    else null,
```

Comment:
Käänsin ehdon toisin päin, sillä pidän henk.koht. enemmän "else null" -tyylisestä tavasta